core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -33,6 +33,9 @@
         <div class="average-ratings stats-card">
           <p class="stat-description">Average Rating</p>
           <h1 class="stat-value"><[dashboardStats.average_ratings || 'N/A']></h1>
+          <p ng-hide="!dashboardStats.num_ratings">
+            (by <[dashboardStats.num_ratings]> users)
```

Comment:
This should say "by 1 user" and not "by 1 users". I think the best way to approach this is to use the plural tag when putting this into translations.